cfg_VHcc_ZNuNu.py

```diff
@@ -37,45 +37,45 @@
     f"{localdir}/datasets/Run2UL2017_MC_VJets.json",
     f"{localdir}/datasets/Run2UL2017_MC_OtherBkg.json",
     f"{localdir}/datasets/Run2UL2017_DATA.json",
+    f"{localdir}/datasets/Run2UL2017_Signal.json"
 ]
 files_2018 = [
     f"{localdir}/datasets/Run2UL2018_MC_VJets.json",
     f"{localdir}/datasets/Run2UL2018_MC_OtherBkg.json",
     f"{localdir}/datasets/Run2UL2018_DATA.json",
 ]
-files_Run3 = [
-    f"{localdir}/datasets/Run3_MC_VJets.json",
-    f"{localdir}/datasets/Run3_MC_OtherBkg.json",
-    f"{localdir}/datasets/Run3_DATA.json",
-]
```

Comment:
Please, put back the Run3 samples lists

Reply:
Put them in place. They are working fine